### PR TITLE
Fix memory leak when using BdvHandlePanel

### PR DESCRIPTION
**Pull request by maarzt**

### CODE REVIEW

**Comment by tpietzsch** [comment]
Thank you!

**Comment by tpietzsch** [comment]
released in bigdataviewer-vistools-1.0.0-beta-10

File: pom.xml [hunk 1]
```diff
@@ -131,5 +131,9 @@
 			<artifactId>ui-behaviour</artifactId>
 			<version>1.3.0</version>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
```
Comment (imagejan) [inline pom.xml]: Should be `<scope>test</scope>`, no?

**Comment by tpietzsch** [comment]
@imagejan good catch, thanks! I fixed it

**Pull request merged**